### DISCUSSION TITLE
Center of side "constraint"

### DISCUSSION THREAD

**Discussion by MonneratRJ** [Ondsel-Development/FreeCAD#110] [Ideas]
I am considering trying Ondsel, I'm a beginner at CAD, my interest is mainly for 3D printing.  But one feature that got me upset is the lack of a centering option on a side... To be honest I, as a maker and not an engineer, I think I speak for many people on the 3D printing hobby, seek a software with good ease to use like sketchup but powerful as freecad

**Comment by prokoudine** [comment]
Sorry, just have noticed this post. We introduced midpoint constraints in the May release. Please see here for more information: https://ondsel.com/blog/ondsel-es-2024-2/#further-sketcher-improvements
